.github/workflows/e2ev2.yaml

```diff
@@ -0,0 +1,54 @@
+name: E2E Version 2 Test
```

Comment:
can we rename to generate e2e matrix or something similar?

Reply:
done